test/unit/person_test.rb

```diff
@@ -1407,4 +1408,327 @@ def test_updated_not_changed_when_adding_notifiee_info
     assert_equal [project, project2, project3], person.projects.sort_by(&:id)
     assert_equal [project,project2], person.administered_projects.sort_by(&:id)
   end
+
+  test 'merge simple attributes' do
+    person_to_keep = FactoryBot.create(:min_person)
+    other_person = FactoryBot.create(:max_person)
+    other_person_attributes = {}
+    simple_attributes.each do |attribute|
+      other_person_attributes[attribute] = other_person.send(attribute)
+    end
+
+    disable_authorization_checks { person_to_keep.merge(other_person) }
+    person_to_keep.reload
+
+    assert_equal 'Minimal', person_to_keep.last_name
+    assert_equal 'minimal_person@email.com', person_to_keep.email
+    attributes = simple_attributes - %i[last_name email]
+    attributes.each do |attribute|
+      assert_equal other_person_attributes[attribute], person_to_keep.send(attribute),
+                   "Should copy #{attribute} if empty in person to keep"
+    end
+  end
+
+  test 'merge annotations' do
+    person_to_keep = FactoryBot.create(:min_person)
+    person_to_keep.annotate_with(%w[golf merging], 'expertise', person_to_keep)
+    person_to_keep.annotate_with(%w[merger], 'tool', person_to_keep)
+    person_to_keep.save!
+    person_to_keep.reload
+    orig_annotations = {}
+    annotation_types.each do |annotation_type|
```

Comment:
don't use `annotation_types` that is being inherited from PersonMerge. Be explicit

or have a separate test that checks what this method returns

Reply:
Resolved in [8d64fe9](https://github.com/seek4science/seek/pull/1683/commits/8d64fe9b7d6bd86c3eacb81547989a27ec9eb878).